src/isomorphic/ReactDebugTool.js

```diff
@@ -58,17 +63,40 @@ var ReactDebugTool = {
   onSetState() {
     emitEvent('onSetState');
   },
-  onMountRootComponent(internalInstance) {
-    emitEvent('onMountRootComponent', internalInstance);
+  onSetDisplayName(debugID, displayName) {
+    emitEvent('onSetDisplayName', debugID, displayName);
+  },
+  onSetIsEmpty(debugID, isEmpty) {
+    emitEvent('onSetIsEmpty', debugID, isEmpty);
```

Comment:
I added this so the tree devtool would filter them out in `getChildIDs`. Alternatively I can go back to the old approach:

* Make composite check for whether the rendered component is empty, and ignore it in `onSetChildren`
* The tree devtool becomes unaware of parent-child association between composite and empty component so it can't clean it up on unmount easily
* We have to add the native container IDs back so it's cleaned up on native container unmount instead

It feels like what I have now is simpler in terms of logic (all cleanup paths are the same and are caused by unmounting) but I can bring back native container tracking if it is more desirable.